src/pages/guidelines/style-models/style-model-list.mdx

```diff
@@ -0,0 +1,217 @@
+
+<br/>
+
+
+
+| User intent | Description |
+| :---| :--- |
+| [&#x21b3;&nbsp;Editorial&nbsp;navigation](#editorial-navigation) | The user needs to scan available navigation options along with relevant content, identify the right destination and navigate to it. |
+| [&#x21b3;&nbsp;Index&nbsp;navigation](#index-navigation) | The user needs to scan available navigation options, identify the right destination and navigate to it. |
+| [&#x21b3;&nbsp;Listing](#listing) | The user needs to see a list(s) of available options, identify the right destination and navigate to it. |
+| [&#x21b3;&nbsp;Reading](#reading) | The user needs to read long-form content in order to gain in-depth understanding of a subject. |
+| [&#x21b3;&nbsp;Information&nbsp;gathering](#information-gathering) | The user needs to gather and understand concise pieces of information towards making a decision on purchase or subscription. |
+| [&#x21b3;&nbsp;Overview (dashaboard)](#overview) | The user needs to have an overview of a given context or system and identify action items or priority items. |
+| [&#x21b3;&nbsp;Simple&nbsp;transaction](#simple-transaction) | The user needs to submit a small set of information such as user ID and password to the system.  |
+| [&#x21b3;&nbsp;Transaction](#transaction) | The user needs to interact with the system and exchange complex information through multiple steps. |
+| [&#x21b3;&nbsp;Conversantional Help](#conversational-help) | The user needs a way to connect with IBM reps to get support or start a conversation. |
+
+## Editorial navigation
+
+The **editorial navigation style model** is usually used on navigagtional pages that have additional content pieces or provide highly curated sections such as featured products.
+
+<Title>User intent</Title>
+
+The user needs to scan available navigation options along with relevant content, identify the right destination and navigate to it.
+
+<Row>
+<Column colMd={12} colLg={12}>
+
+![Editorial nav style model](./images/editorial-nav-style-models.png)
+
+</Column>
+</Row>
+
+<Title>Examples</Title>
+
+Home page, topic landing page
+
+<br/>
+
+
+
+## Index navigation
+
+Compared to the editorial navigation style model, the **index navigation style model** is for navigational pages that mainly focus on helping the user find the right destination without the help of additional content.
+
+<Title>User intent</Title>
+
+The user needs to scan available navigation options, identify the right destination and navigate to it.
+
+
+<Row>
+<Column colMd={12} colLg={12}>
+
+![Index navigation style model](./images/index-nav-style-models.png)
+
+</Column>
+</Row>
+
+<Title>Examples</Title>
+
+Category listing page, product listing page
+
+
+
+## Listing
```

Comment:
Should this be "Listing navigation" ?

Reply:
At the moment, it's called simply listing. We can revisit this post-launch.